clippy_lints/src/utils/conf.rs

```diff
@@ -34,6 +34,7 @@ const DEFAULT_DOC_VALID_IDENTS: &[&str] = &[
     "CamelCase",
 ];
 const DEFAULT_DISALLOWED_NAMES: &[&str] = &["foo", "baz", "quux"];
+const DEFAULT_ALLOWED_IDENTS_BELOW_MIN_CHARS: &[&str] = &["i", "j", "x", "y", "z", "w", "n"];
```

Comment:
For list configurations with default parameters, it's nice to allow the user to only extend the list and not replace it. Clippy has a nice small util to support the `".."` value to indicate this behavior.

Here is a link to the relevant code snippet:

https://github.com/rust-lang/rust-clippy/blob/d44ea7c5c5d51403baa6eeb0e3bbdb0a1ffeed1f/clippy_lints/src/utils/conf.rs#LL586C1-L587C104

Could you also update the conf documentation, to mention this option? You can just copy that part from the other configs :)

Reply:
Done